src/pages/formFields.js

```diff
@@ -0,0 +1,109 @@
+export const ScholarshipFields = {
+  name: {
+    key: 1,
```

Comment:
`key` is not needed here. The `key` in list elements can simply be the key here (`name`)